### PR TITLE
Program redesign

### PR DESCRIPTION
**Pull request by j0nas**
# Program redesign

## Intent

Implements the redesigned `programSection` per updated Figma sketches.

Solves [this Shortcut Story](https://app.shortcut.com/sanity-io/story/16054/program-page-front-end).

## Details

This PR adds a semi-hardcoded `program?venue=<VENUE_NAME>` entry to _[[...slug]].tsx_' `getStaticPaths` for each Venue referenced by a Program. This currently correlates with the fact that the `programsSection` is rendered at _/program_, but if the program is ever moved to another location, this pre-generation will break; hence "semi-hardcoded".

## Testing this PR

1. Go to [/program](https://structured-content-2022-web-git-program-redesign.sanity.build/program) for [this PR's Preview Deploy URL](https://structured-content-2022-web-git-program-redesign.sanity.build)
2. Verify that the rendered elements correspond with the [Figma sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=7958%3A22278) for all specified viewports
3. Verify that you can select other venues from the "VenueNav" component, and that switching between Programs for the various Venues happens quickly (indicating that the various Program pages are pre-generated)

### CODE REVIEW

File: web/components/VenueNav/VenueNav.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+interface VenueNavProps {
+  venues: Venue[];
+  activeVenue?: Venue;
+}
+
+const venueLabel = (venueName: string) =>
+  ({
+    'San Francisco': 'Main venue',
+    Virtual: 'Online',
+  }[venueName] || 'Satellite');
```
Comment (oyvind-stenhaug) [inline web/components/VenueNav/VenueNav.tsx:17]: Is there a less hacky way to classify the `'Main venue'` here?

Previously we've looked at the first venue in the event document, I think, but I suppose another approach could be to hard-code a `mainVenueId` in a similar way to the existing `mainEventId`. Don't know which is easier.

**Review by oyvind-stenhaug — CHANGES_REQUESTED** [review]
Tried to review… just one comment about the hardcoded labels.

Maybe remove the "Draft" status from the PR once you've reviewed my changes and/or we have received a go-ahead from Sanity?

File: web/components/VenueNav/VenueNav.tsx (hunk 1 above)
Comment (j0nas) [reply web/components/VenueNav/VenueNav.tsx]: What do you think of 60569da? It's the best I could come up with given the circumstances.

**Review by wilhelm — APPROVED** [review]
HTML/CSS looks good to me.

**Pull request merged**